docs/src/pages/components/date-picker/ResponsiveDatePickers.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import {
+  MobileDatePicker,
+  DesktopDatePicker,
+  DatePicker,
+} from '@material-ui/lab/DatePicker';
+
+export default function DatePickersVariants() {
+  const [value, setValue] = React.useState<Date | null>(new Date());
+
+  return (
+    <div style={{ width: 300 }}>
+      <MobileDatePicker
+        clearable
+        label="For mobile"
+        inputFormat="MM/dd/yyyy"
+        toolbarPlaceholder="Enter Date"
+        value={value}
+        onChange={(newValue) => setValue(newValue)}
+        renderInput={(props) => <TextField {...props} margin="normal" />}
```

Comment:
The demos should be singled focused

```suggestion
        label="For mobile"
        value={value}
        onChange={(newValue) => setValue(newValue)}
        renderInput={(props) => <TextField {...props} margin="normal" />}
```

Also, ideally we would have the value uncontrolled:

```suggestion
        label="For mobile"
        renderInput={(props) => <TextField {...props} margin="normal" />}
```